content/en/agent/configuration/dual-shipping.md

```diff
@@ -327,6 +327,33 @@ DD_NETWORK_DEVICES_METADATA_ADDITIONAL_ENDPOINTS="[{\"api_key\": \"apiKey2\", \"
 
 {{% agent-dual-shipping %}}
 
+## Network Path
+
+### YAML configuration
+
+Requires Agent >= 6.55 or 7.55.
```

Comment:
I'm confused about the inclusion of version 7.55, since it is greater than 6.55 and is therefore already included in the phrase "Agent >= 6.55". Is there a reason to explicitly include 7.55?

Reply:
@urseberry Currently, when a new version of Datadog Agent is release, we always release 7.x.x and 6.x.x.

For example, we are currently releasing 6/7.55 (6.55 and 7.55), the next version of the agent will be 6/7.56 (6.56 and 7.55).

So "Requires Agent >= 6.55 or 7.55." means: 
- if user is using 6.X, they need at least 6.55
- if user is using 7.X, they need at least 7.55

There are some additional explanation here:
https://docs.datadoghq.com/agent/versions/

Let me know if that helps.

Reply:
Yes, I understand. Thank you!